ddtrace/settings/_config.py

```diff
@@ -493,9 +493,10 @@ def __init__(self):
         # Report Telemetry for Agent Connection Configurations. We need to do this here to avoid circular imports
         from ddtrace.internal.agent import config as agent_config
 
-        from ._telemetry import report_telemetry
+        # TODO: remove this import from here
+        from ddtrace.internal.telemetry import report_configuration
 
-        report_telemetry(agent_config)
+        report_configuration(agent_config)
```

Comment:
Can we move this function call to `ddtrace.internal.agent`

Reply:
Sure 👍  Where in the agent do you reckon would be a good place to make this call?

Reply:
We can call this function right after `ddtrace.internal.agent.config` is initialized. Similar to how we handle the other envier config objects